src/three/TilesRenderer.js

```diff
@@ -566,7 +563,6 @@ export class TilesRenderer extends TilesRendererBase {
 				break;
 
 			case 'z':
-				upAdjustment.identity();
```

Comment:
Thanks for the clean up! Why don't we remove the `case 'z':` block in this case. I think leaving an empty case will look like a mistake in the future.

Reply:
@gkjohnson Fixed